input/pagecontent/changes.md

```diff
@@ -7,6 +7,14 @@ This change log documents the significant updates and resolutions implemented fr
 
 #### Changes in this version
 - New profiles: 
+<<<<<<< HEAD
```

Comment:
It seems that a conflict was not resolved properly. Please update.

Reply:
no conflicts now?